server/src/services/EmailServices.js

```diff
@@ -225,6 +225,63 @@ const EmailServices = {
 
     return SES.send(command);
   },
+
+  async sendStreamAttachedEmail(html, text, subject, streams, toAddresses, fromAddress) {
+    const msg = mimemessage.factory({
+      contentType: 'multipart/mixed',
+      body: [],
+    });
+    msg.header('From', fromAddress);
+    msg.header('To', toAddresses);
+    msg.header('Subject', subject);
+    const alternateEntity = mimemessage.factory({
+      contentType: 'multipart/alternate',
+      body: [],
+    });
+
+    const htmlEntity = mimemessage.factory({
+      contentType: 'text/html;charset=utf-8',
+      body: html,
+    });
+
+    const plainEntity = mimemessage.factory({
+      body: text,
+    });
+
+    alternateEntity.body.push(htmlEntity);
+    alternateEntity.body.push(plainEntity);
+
+    msg.body.push(alternateEntity);
+
+    for (const stream of streams) {
+      const attachment = mimemessage.factory({
+        contentType: mime.getType(), // get mimetype of stream
```

Comment:
```suggestion
        contentType: 'application/pdf'
```

the MIME type of the stream is `application/pdf` since that is the data inside it.